src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -114,7 +114,10 @@ static ExperimentalFeature()
                     description: "New parameter set for ForEach-Object to run script blocks in parallel"),
```

Comment:
@bergmeister, your last commit had 1 failures in `PowerShell-CI-linux`
Validate Update-Help from the Web for all PowerShell modules for user scope..Validate Update-Help for module 'Microsoft.PowerShell.Host' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```

Reply:
Sporadic failure, went away after a retry